spring-kafka/src/main/java/org/springframework/kafka/support/Acknowledgment.java

```diff
@@ -85,4 +85,7 @@ default boolean isOutOfOrderCommit() {
 		return false;
 	}
 
+	default boolean isAnyManualAck() {
```

Comment:
I don't think this method makes sense since `Acknowledgment` abstraction is there exactly for the manual acks.
See the logic in the `doInvokeOnMessage()`.
What is your exact motivation do introduce this option over here?
Thanks

Reply:
As you said, `Acknowledgment abstraction is there exactly for the manual acks` is true! 
i reverted this method `isAnyManualAck()`.